modular_skyrat/modules/exp_corps/code/clothing.dm

```diff
@@ -0,0 +1,179 @@
+	worn_icon = 'modular_skyrat/modules/exp_corps/icons/mob_backpack.dmi'
+	slowdown = 0
+	resistance_flags = FIRE_PROOF
+
+/obj/item/clothing/suit/armor/vest/expeditionary_corps
+	name = "expeditionary corps armor vest"
+	desc = "An armored vest that provides decent protection against most types of damage."
+	icon_state = "exp_corps"
+	icon = 'modular_skyrat/modules/customization/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/modules/customization/icons/mob/clothing/suit.dmi'
+	body_parts_covered = CHEST|GROIN|ARMS
+	armor = list(MELEE = 60, BULLET = 60, LASER = 60, ENERGY = 60, BOMB = 60, BIO = 0, RAD = 20, FIRE = 80, ACID = 100, WOUND = 30)
+	cold_protection = CHEST|GROIN|ARMS
+	heat_protection = CHEST|GROIN|ARMS
+	dog_fashion = null
+	allowed = list(
+		/obj/item/melee,
+		/obj/item/ammo_box,
+		/obj/item/ammo_casing,
+		/obj/item/flashlight,
+		/obj/item/gun,
+		/obj/item/kitchen/knife,
+		/obj/item/reagent_containers,
+		/obj/item/restraints/handcuffs,
+		/obj/item/tank/internals/emergency_oxygen,
+		/obj/item/tank/internals/plasmaman
+		)
+/obj/item/clothing/suit/space/hardsuit/expeditionary_corps
+	name = "expeditionary corps hardsuit"
+	desc = "An advanced hardsuit designed for exploratory missions."
+	icon = 'modular_skyrat/modules/customization/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/modules/customization/icons/mob/clothing/suit.dmi'
+	icon_state = "hardsuit-hexp_corps"
+	inhand_icon_state = "eng_hardsuit"
+	hardsuit_type = "hexp_corps"
+	armor = list(MELEE = 60, BULLET = 60, LASER = 60, ENERGY = 60, BOMB = 60, BIO = 0, RAD = 20, FIRE = 80, ACID = 100, WOUND = 30)
+	allowed = list(/obj/item/gun, /obj/item/ammo_box,/obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/melee/transforming/energy/sword/saber, /obj/item/restraints/handcuffs, /obj/item/tank/internals)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/expeditionary_corps
+	jetpack = /obj/item/tank/jetpack/suit
+	cell = /obj/item/stock_parts/cell/hyper
+	slowdown = 0.5
+	hardsuit_tail_colors = list("434", "223", "988")
+
+/obj/item/clothing/head/helmet/space/hardsuit/expeditionary_corps
+	name = "expeditionary corps hardsuit helmet"
+	desc = "An advanced hardsuit helmet designed for exploratory missions."
+	icon = 'modular_skyrat/modules/customization/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/modules/customization/icons/mob/clothing/head.dmi'
+	icon_state = "hardsuit0-hexp_corps"
+	inhand_icon_state = "sec_helm"
+	hardsuit_type = "hexp_corps"
+	armor = list(MELEE = 60, BULLET = 60, LASER = 60, ENERGY = 60, BOMB = 60, BIO = 0, RAD = 20, FIRE = 80, ACID = 100, WOUND = 30)
+	visor_flags_inv = HIDEMASK|HIDEEYES|HIDEFACE|HIDEFACIALHAIR|HIDESNOUT
+	visor_flags = STOPSPRESSUREDAMAGE
+	slowdown = 0.5
+
+/obj/item/clothing/head/helmet/expeditionary_corps
+	name = "expeditionary corps helmet"
+	desc = "A robust helmet worn by Expeditionary Corps troopers. Alt+click it to toggle the NV system."
```

Comment:
Tbh I'd like this on an action button, even if people dislike them. It's easier to access in combat